dev-guide.md

```diff
@@ -111,7 +130,27 @@ source venv_smallboard/bin/activate
 (venv_smallboard)$ python manage.py migrate
 ```
 
-#### Copying production data to local database
+#### Local deployment
+
+Once the Python environment and database are set up and running, you can run Small Board locally using
+
+```
+(venv_smallboard)$ python manage.py runserver
+```
+
+You can view the app in your browser at [http://127.0.0.1:8000/]().
```

Comment:
Maybe we can use the markdown shortcut `<http://...>` for these kinds of links? https://daringfireball.net/projects/markdown/syntax#autolink

Reply:
Sure (I didn't change this, just reshuffled it a bit)